src/components/HomeSkeleton.jsx

```diff
@@ -0,0 +1,28 @@
+import { Flex } from "@sparrowengg/twigs-react";
+const HomeSkeleton = () => {
+  return (
+    <Flex
+      justifyContent="center"
+      css={{
+        flexWrap: "wrap",
+        gap: "$10",
+        overflow: "scroll",
+        padding: "$2",
+        width: "100%",
+      }}
+    >
+      {new Array(20).fill("").map((ele, i) => (
```

Comment:
@arun-121 by default are we setting it to 20 elements????

Reply:
changed to  a constant variable instead of hardcoding